packages/@eris-boiler/common/src/modules/stores/load-map.ts

```diff
@@ -44,19 +78,25 @@ export abstract class LoadMap<T> extends ExtendedMap<string, T> {
     const file = await fs.stat(path)
     const files = file.isDirectory() ? await fs.readdir(path) : [ '' ]
     const res = []
+
     for (const fd of files) {
       const filePath = join(path, fd)
+      const importName = /^(?!.*\.(?:d|spec|test)\.ts$).*(?=\.ts$)/.exec(filePath)?.[0]
```

Comment:
It's just my personal preference, but I find it cleaner to define Regex as constants near the top of the file

Reply:
probably we should have some constants file defined somewhere, i will check if there are other candidates